Resources/Prototypes/_Goobstation/tags.yml

```
@@ -325,4 +325,15 @@
+  
+- type: Tag
+  id: Bomb
+
+- type: Tag
+  id: Rifle
 
+- type: Tag
+  id: Hacking
+
+- type: Tag
+  id: Jetpack
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)